dnf-json

```diff
@@ -200,6 +200,7 @@ class Solver():
                 "summary": package.summary,
                 "description": package.description,
                 "url": package.url,
+                "remote_location": package.remote_location(),
```

Comment:
In the `depsolve()` call we use `repo_id` to match packages to repositories. Should we do the same thing here?

Reply:
Thanks @teg for the hint, absolutely we can use that to identify back the package repository, especially we are providing that repository ids.

Updated this PR to reflect that.

Reply:
Thanks!